File: MISC/CodeVita4/Round2/Sort.py
t=int(input())
while t>0:
	t-=1
	st=input().split()
	k=int(input())
	for i in range(0,len(st)):
		if i%2==0:
			st[i]=st[i].lower()
		else:
			st[i]=int(st[i])
	#print(st)
	for i in range(k):
		cur=st[i]
		for j in range(i+2,int(len(st)),2):
			if cur>st[j]:
				cur=st[j]
				pos=j
		if cur!=st[i]:
			j=pos
			temp=st[i]
			st[i]=st[pos]
			st[pos]=temp

	for i in st:
		print(i,end=' ')
	print()

File: MISC/CodeVita4/Round2/Sort.cpp
#include<bits/stdc++.h>

using namespace std;

struct node{
	char str[100];
	int n;

};

int sort(node arr[],int top,int k){
	int pos,temp,min;
	node cur;
	char temp[100];
	for(int i=0;i<k;i++){
		cur[i].str=
	}
}

int main()
{
   	int t,n;
    scanf("%d",&t);
    while(t--)
    {	char str[100];
    	node arr[10000];
    	int top=-1;
    	char ch='a';
    	int num;
    	scanf("%d",&n);
    	while(ch){
    		scanf("%s %d",str,&num);
    		arr[++top].str=str;
    		arr[top].n=num;
    		ch=getchar();
    	}
    	sort(arr,top,n);
    }

    return 0;
}
